app/modules/pullRequests/actions/apiSaga/apiPost.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import validate from 'lib/validate';
+
+import * as a from '../../actionTypes';
+
+const apiPost = (
+  message: string,
+  sourceTopicId: string,
+  targetTopicId: string,
+  userId: string,
+): a.ApiPostAction => {
+  const validatedPayload = validate.stringProps(
+    ['message', 'sourceTopicId', 'targetTopicId', 'userId'],
```

Comment:
Minor nitpick: ids don't strictly need stringprop validation since they're not user input and will realistically never be an empty / untrimmed string.